app/assets/javascripts/slices/app/helpers/i18n.js

```diff
@@ -0,0 +1,5 @@
+Handlebars.registerHelper('i18n', function(options) {
+  if (slices.i18n) {
+    return options.fn(this);
+  }
```

Comment:
I'll add an `else` here in the usual handlebars way, so caller can do something if i18n is _not_ enabled.